website_sale_affiliate/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
+
+=================
+Affiliate Program
+=================
+
+This module allows for creating affiliates and track their conversions.
```

Comment:
This module allows for the creation of affiliates and the tracking of sales conversions generated by them.